### PR TITLE
Octotree Additions

### PR DESCRIPTION
**Pull request by JanKoppe**
#### What's the issue:
Octotree Addon was still displayed with very bright white Background.

#### What's fixed:
Added CSS rules to set colors to the ones used in github-dark

#### Screenshots:
before
after

Styles for the Octotree Addon (https://github.com/buunguyen/octotree)

### CODE REVIEW

File: github-dark.css [hunk 1]
```diff
@@ -783,3 +783,30 @@ hr,
     background-color: #292c36;
     border: 1px solid #4c4d55;
 }
```
Comment (cquanu) [inline github-dark.css]: Can you make a comment here? /* Octotree Addon */

File: github-dark.css [hunk 2]
```diff
@@ -783,3 +783,30 @@ hr,
...
+.octotree_github_sidebar .octotree_views .octotree_view .octotree_view_header {
+    background-color: #292c36;
+    border-bottom: 1px solid #4c4d55;
+    background-image: none;
+}
+.octotree_views .octotree_view .octotree_view_header {
+    text-shadow: none;
+}
+.octotree_github_sidebar input, .octotree_github_sidebar textarea {
+ 	background-color: #101420;
```
Comment (cquanu) [inline github-dark.css]: Double indent

File: github-dark.css [hunk 3]
```diff
@@ -783,3 +783,30 @@ hr,
...
+    border: 1px solid #4c4d55;
+    padding: 2px;
+    color: white;
```
Comment (cquanu) [inline github-dark.css]: Use hex code instead

**Comment by cquanu** [comment]
Hi thanks for the help. I commented some line notes. Do check them out. Thanks!

File: github-dark.css (hunk 3 above)
Comment (cquanu) [inline github-dark.css]: Use Hex Code instead

**Comment by JanKoppe** [comment]
Oh yes, overlooked those in the hasty commit. should be fixed now, sorry. Also, I can't seem to get the symbols in the buttons (upper right) to be displayed in white. I would probably have to override them in the CSS file, as they are SVGs. Not sure if this is a good idea. What do you think?

**Comment by cquanu** [comment]
@JanKoppe You can do a CSS invert filter

**Comment by cquanu** [comment]
<pre><code>-webkit-filter: invert(1);
filter: invert(1);
</code></pre>

File: github-dark.css [hunk 4]
```diff
@@ -796,17 +797,17 @@ hr,
     text-shadow: none;
 }
 .octotree_github_sidebar input, .octotree_github_sidebar textarea {
- 	background-color: #101420;
+    background-color: #101420;
     border: 1px solid #4c4d55;
     padding: 2px;
-    color: white;
+    color: #ffffff;
```
Comment (cquanu) [inline github-dark.css]: color: #fff;

**Comment by JanKoppe** [comment]
That seems to work okay.

**Comment by cquanu** [comment]
@JanKoppe Looking good 👍

**Comment by JanKoppe** [comment]
Right, should have seen that. Sorry.

**Comment by cquanu** [comment]
It's fine :) Ready to merge?

**Comment by JanKoppe** [comment]
Yep, looks good to me!

**Pull request merged**